CMSgov/design-system
# [WNMGDS-2052] temporary 5.0 rel solution for analytics in propTable

## Summary

- Updates the `PropTable` doc site with an additional optional `hasAnalytics` prop that displays the analytics props currently stored in a separate file.
- Updates component doc mdx where these props are utilized
- **NOTE**: This is a bandaid fix that will need to be resolved before v6.0 is released in order to have these props available in the documentation due to the limitations of `react-docgen` and the `gatsby-transformer-react-docgen` plugin.

### How to test:

- `yarn clean && yarn install && yarn build && yarn build:storybook:docs && yarn start` - view changed pages, confirm analytics props appear at top of prop table
- `yarn test`
